package/src/types/v3-matches.ts

```diff
@@ -0,0 +1,232 @@
+import type { Characters, Mode, Rank, Region, ValorantMap, Weapon } from "./general.js"
```

Comment:
Why are you importing javascript files here?

Also this files doesn't even exist.